Removes most of the Torch's Cameras

juxjux9930 opened the pull request:
🆑 Jux
maptweak: Most of the Torch's cameras have been removed. Areas exempt are generally hallways, main ship entrance points, shuttle cockpits, and high traffic or high security areas.
maptweak: Some cameras have had their positions adjusted.
/🆑 

Why? The short answer is antag freedom. Cameras covering almost every inch of the ship meant that the only truly private places were few and far between, especially since removing a camera triggered an alarm. Removing this many cameras may seem excessive, but they were by and large for private rooms like offices, meeting areas, hell, the captain's BEDROOM. I don't believe this is uncalled for.

Some areas, like mess or hydroponics, went from having multiple cameras to one or two- in these cases, I tried to resposition the remaining ones for better coverage.

Shuttles were a tough call for me. I ended up removing all cameras from the Guppy, and most from the Aquila and Charon. Aquila kept the cockpit and crew area, and Charon kept the cockpit and primary airlock.

DrDiasyl commented:
I think Torch had this many camera's because of the AI (so it can see), but now as AI is gone this won't hurt **alot** but still will hurt.

Torch is a pretty important government vessel, I think camera's being everywhere was fine from this logical perspective.

juxjux9930 commented:
From a logical perspective, a lot of things on the torch don’t really make sense. I think it’s important to have some illogical things for a good game- and the increased suspense will, in my opinion, improve things.

SierraKomodo commented:
> I think Torch had this many camera's because of the AI (so it can see), but now as AI is gone this won't hurt **alot** but still will hurt.
> 
> Torch is a pretty important government vessel, I think camera's being everywhere was fine from this logical perspective.

Gameplay is going to trump realism here - Removal of excess cameras was already a planned thing, to reduce the amount of camera-hopping catching random antagonists.

Commit:
Removes most of the Torch's cameras

Commit:
Moves most helmet cams to a Helmet cam network

juxjux9930 commented:
Updated with even more files touched!

Re-did what cams were on what networks, and removed torch networks that weren't in use.
Bridge & Command networks have been merged, petrov cameras have been placed into science network, hangar cams into d5.

 Commit #2 moves helmet cameras all to a new helmet camera network, except for the apparently existing merc & ert ones.

cdbrunow commented:
It looks like you may have fat-finger deleted a table in primary tool storage?
![tablemissing](https://github.com/Baystation12/Baystation12/assets/32042876/ca647882-5bb7-4ccf-86c7-a9f0cb72ea36)

juxjux9930 commented:
> It looks like you may have fat-finger deleted a table in primary tool storage? ![tablemissing](https://user-images.githubusercontent.com/32042876/247840285-ca647882-5bb7-4ccf-86c7-a9f0cb72ea36.png)

wow, sure did. fixed

Spookerton reviewed (approved):
https://youtu.be/rYE84KgOPXY

Nerezza commented on code/modules/modular_computers/file_system/programs/generic/camera.dm:
```diff
@@ -20,6 +20,8 @@
 		if(NETWORK_THUNDER)
 			return 0
+		if(NETWORK_HELMETS)
+			return access_eva
```
Late critique here but only EMTs and the CMO have this access in medical, you've removed the ability for doctors to see the camera on the EMTs under their command.
